spec/system/organization_system_spec.rb

```diff
@@ -116,6 +117,20 @@
         expect(page).to_not have_content('Media Information')
         expect(@organization.reload.partner_form_fields).to eq([])
       end
+
+      it "can disable if the org does NOT use single step invite and approve partner process" do
+        choose("organization[use_single_step_invite_and_approve_partner_process]", option: false)
+
+        click_on "Save"
+        expect(page).to have_content("No")
+      end
+
+      it "can enable if the org uses single step invite and approve partner process" do
+        choose("organization[use_single_step_invite_and_approve_partner_process]", option: true)
+
+        click_on "Save"
+        expect(page).to have_content("Yes")
+      end
```

Comment:
There should be a request spec that checks the correct button being shown on the partners page based on the value of the flag.

Reply:
Added